artifacts/linting_rules/lint_function/camara-security-no-secrets-in-path-or-query-parameters.js

```diff
@@ -0,0 +1,23 @@
+const sensetiveData = ['MSISDN','IMSI'];
```

Comment:
Correct spelling is "sensitive"

Reply:
Should we add further values to the array, e.g. 'phoneNumber' which we use instead of 'MSISDN'?

Reply:
Yes, we can add further value in array. @hdamker If you think we need to replace the MSISDN with phoneNumber then I will update the same in PR

Reply:
I would add `phoneNumber` but not replacing `MSISDN`.

Reply:
> I would add `phoneNumber` but not replacing `MSISDN`.

Updated suggested changes.

Reply:
Current API Design Guidelines states: 

> The classification of data tagged as sensitive should be assessed for each API project, but might include the following examples:
> 
>  -   phone number (MSISDN) must be cautiously handled as it is not solely about the number itself, but also knowing something about what transactions are being processed for that customer
> -    localisation information (such as latitude & longitude) associated with a device identifier as it allows the device, and hence customer, location to be known
>  -   physical device identifiers, such as IP addresses, MAC addresses or IMEI

But we can here think of commonly used names for sensitive data

Reply:
In our camera system, we do not use "MSISDN" as the identifier for a phone number; instead, we use the term "phoneNumber."
For instance, here's an example of device information:
{
  "device": {
    "phoneNumber": "123456789",
    "networkAccessIdentifier": "123456789@domain.com",
    "ipv4Address": {
      "publicAddress": "84.125.93.10",
      "publicPort": 59765
    },
    "ipv6Address": "2001:db8:85a3:8d3:1319:8a2e:370:7344"
  },
  "maxAge": 60
}

Furthermore, in the context of the "camara-language-avoid-telco.js" rule, we've incorporated a substitution rule that changes "MSISDN" to "Phone Number." So, it would be illogical to include "MSISDN" in the list of sensitiveData since this term will already be replaced with "Phone Number" by the "camara-language-avoid-telco.js" rule.

Reply:
@sachinvodafone Completely correct, using MSISDN within path or query parameters would violate two rules at the same time ... but that does not mean that a new sub project will not do this mistake initially ;-) But you are right, to avoid that people think that MSISDN is valide parameter we can also replace/omit it here.

But I guess camara-language-avoid-telco.js will not "substitute" `MSISDN` automatically but just raising a warning and a recommendation to use phone number instead, right?

Reply:
@hdamker @sachinvodafone **Yes Right**
**camara-language-avoid-telco.js** will not "substitute" MSISDN automatically but just raising a warning and a recommendation to use phone number instead.